### PR TITLE
Web branch

### PR DESCRIPTION
**Pull request by Chike1990**
I cloned an article page of the New York Times applying the concepts I learned over the week. Specifically,
I first created a repository on Github, created a branch from it, added ReadMe file, added linter checks, index.html file, and style.css file.
The structure and content of the New York Times article page were laid out in HTML- using HTML 5 semantic tags; the images were downloaded from the NYT page and were included in the cloned web page; these contents were style using CSS - and by applying concepts like positioning, flex, float, etc.
In other words, this page includes a navbar, an aside, main, sections, articles, divs, footer, etc. Prior to sending a pull request, linters were checked for errors in the style and index files and about five code reviewers reviewed this code and made recommendations that have been included.

### CODE REVIEW

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o:  
Great job on this project.
Semantic HTML has been used throughout the page.
Let's fix one issue before we proceed.

### REQUIRED
The bottom part of the page containing more articles is missing. See this :point_down: 
Original Page | Cloned Page
|-------------------|------------------|
Screenshot from 2021-02-25 13-33-03 | Screenshot from 2021-02-25 13-44-36

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Comment by Chike1990** [comment]
Alright, thank you

Regards,
Chibuike

**Comment by Chike1990** [comment]
I cloned an article page of the New York Times applying the concepts I learned over the week. Specifically,
I first created a repository on Github, created a branch from it, added ReadMe file, added linter checks, index.html file, and style.css file.
The structure and content of the New York Times article page were laid out in HTML- using HTML 5 semantic tags; the images were downloaded from the NYT page and were included in the cloned web page; these contents were style using CSS - and by applying concepts like positioning, flex, float, etc.
In other words, this page includes a navbar, an aside, main, sections, articles, divs, footer, etc. Prior to sending a pull request, linters were checked for errors in the style and index files and about five code reviewers reviewed this code and made recommendations that have been included.

**Review by teekaytech — CHANGES_REQUESTED** [review]
### Status: Changes Required :warning: :vertical_traffic_light: 

**Hello @Chike1990  :wave:**

__Thank you for implementing the changes highlighted by the last reviewer.__

**Major highlights:**
- You did well to add the fixed footer at the bottom of the page :+1: 
- Your webpage looks good: you paid attention to the details :ok: 
- Nice implementation of the `Theory of Inflation` section :1st_place_medal: 
- All linter checks are running successfully! :100: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] The project screenshot is not rendering on your Readme. Kindly look into this. This [stack overflow thread](https://stackoverflow.com/questions/14494747/how-to-add-images-to-readme-md-on-github) might help.
- [ ] The `Issues page` and `MIT license` links on your Readme are broken. Kindly [add the license file](https://stackoverflow.com/questions/31639059/how-to-add-license-to-an-existing-github-project) to your project and ensures that all links lead to the correct pages. 

**[ OPTIONAL CHANGES ]**
_**Note: the highlighted changes in this category are not compulsory: your project will be approved if they're not implemented. However, you're advised to make these changes so as to develop a more professional project.**_
- [ ] I observed that you added the `SVG` image tags directly in your HTML file. This will work, but when it becomes too much, it tends to mess up your HTML file. Instead, you can create separate files for individual SVG images, then use the image tag to reference them in your HTML file. [This article](https://www.freecodecamp.org/news/use-svg-images-in-css-html/) might help.

**That's it!**
* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by Chike1990** [comment]
I cloned an article page of the New York Times applying the concepts I learned over the week. Specifically,
I first created a repository on Github, created a branch from it, added ReadMe file, added linter checks, index.html file, and style.css file.
The structure and content of the New York Times article page were laid out in HTML- using HTML 5 semantic tags; the images were downloaded from the NYT page and were included in the cloned web page; these contents were style using CSS - and by applying concepts like positioning, flex, float, etc.
In other words, this page includes a navbar, an aside, main, sections, articles, divs, footer, etc. Prior to sending a pull request, linters were checked for errors in the style and index files and about five code reviewers reviewed this code and made recommendations that have been included.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CLONED_NYT
+
+> A clone of New York Times article
+
+![screenshot](Assets\ProjectScreenShot.jpg)
```
Comment (umairarshadbutt) [inline README.md]: The image path is wrong, please fix it. You can update from `screenshot` to `screenshot`

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Chike1990 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes 👇.
- [ ] The project screenshot is not rendering on your Readme. Kindly fix this issue as the previous TSE explained. See the Hit in comment
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by Chike1990** [comment]
I cloned an article page of the New York Times applying the concepts I learned over the week. Specifically,
I first created a repository on Github, created a branch from it, added ReadMe file, added linter checks, index.html file, and style.css file.
The structure and content of the New York Times article page were laid out in HTML- using HTML 5 semantic tags; the images were downloaded from the NYT page and were included in the cloned web page; these contents were style using CSS - and by applying concepts like positioning, flex, float, etc.
In other words, this page includes a navbar, an aside, main, sections, articles, divs, footer, etc. Prior to sending a pull request, linters were checked for errors in the style and index files and about five code reviewers reviewed this code and made recommendations that have been included.

**Review by enaburekhan — CHANGES_REQUESTED** [review]
# Status: Required changes:full_moon:
**Hi Chibuike!
Many thanks for implementing the changes requested by the previous reviewers:clap::clap:. The UI is awesome and the cloned website looks very much like the original:100:. Keep up the good job:thumbsup:.
We just need to make an adjustment to that same screen-shot in the readme to get this project approved:muscle:.** 

- [ ] The current screen-shot is good but not good enough to be featured in the readme of a project. check my [screen-shot](https://imgur.com/a/R2EDvEI) from the same cloned project. Use something similar or better.

Make the above-required change and when you are ready, request another review.
Kindly drop your questions :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy hacking
Eric Enaburekhan

**Comment by Chike1990** [comment]
I cloned an article page of the New York Times applying the concepts I learned over the week. Specifically,
I first created a repository on Github, created a branch from it, added ReadMe file, added linter checks, index.html file, and style.css file.
The structure and content of the New York Times article page were laid out in HTML- using HTML 5 semantic tags; the images were downloaded from the NYT page and were included in the cloned web page; these contents were style using CSS - and by applying concepts like positioning, flex, float, etc.
In other words, this page includes a navbar, an aside, main, sections, articles, divs, footer, etc. Prior to sending a pull request, linters were checked for errors in the style and index files and about five code reviewers reviewed this code and made recommendations that have been included.

**Review by oloomoses — APPROVED** [review]
# Congratulation, Project Approved! :tada: :clap: :muscle: 
You have done a great job implementing the changes from the previous review. 
The project is complete and meets all the requirements. Congratulation once again.
You can now merge and close this pull request. Don't forget to submit the completion form after.
I wish you the best in your next project and see you at the top :rocket: 
### Happy coding.
_Slack: Oloo Moses_

**Comment by Chike1990** [comment]
[Sighs] 
Let me go take a cool drink😊 

Thanks,

Chike

**Comment by Chike1990** [comment]
Complete project

**Pull request merged**